### PR TITLE
update docs after node/cli split

### PR DESCRIPTION
**Pull request by chefsale**
update of docs after: https://github.com/calimero-network/core/pull/796/files

### CODE REVIEW

**Review by petarjuki7 — APPROVED** [review]

File: docs/04-build/04-dev-mode.mdx [hunk 1]
```diff
@@ -37,17 +37,20 @@ This should be done in a separate terminal window. For more information, see the
...
+To list all supported commands for the deamon, run the following command:
+
+```bash title="Terminal"
+cargo run -p merod -- help
+```
+
+To list all supported commands for the cli, run the following command:
 
 ```bash title="Terminal"
 cargo run -p meroctl -- help
```
Comment (frdomovic) [inline docs/04-build/04-dev-mode.mdx:49]: Should this be meroctl or updated with merod?

File: docs/03-getting-started/01-setup.mdx [hunk 2]
```diff
@@ -48,15 +48,15 @@ representing the network participants.
 #### Initialize and start coordinator node (separate terminal)
 
 ```bash title="Terminal"
-cargo run -p meroctl -- --node-name coordinator --home data init --server-port 2427 --swarm-port 2527
-cargo run -p meroctl -- --node-name coordinator --home data run --node-type coordinator
+cargo run -p merod -- --node-name coordinator --home data init --server-port 2427 --swarm-port 2527
```
Comment (frdomovic) [inline docs/03-getting-started/01-setup.mdx:51]: Didn't follow through, whats the difference between merod and meroctl?

Comment (chefsale) [reply docs/03-getting-started/01-setup.mdx:51]: Merod is the node, meroctl is the cli. It doesn;t make sense to have them bundled.

Comment (frdomovic) [reply docs/03-getting-started/01-setup.mdx:51]: Okay got it, thanks.

**Pull request merged**